vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,98 +67,79 @@
+          <Annotation Term="Core.Description" String="Kind of temporal collection" />
         </Property>
-        <Property Name="SystemTime" Type="Temporal.GranularityDateTimeOffset">
-          <Annotation Term="Core.Description" String="System time granularity" />
+        <Property Name="PeriodKind" Type="Temporal.PeriodKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of application time period" />
         </Property>
         <Property Name="ObjectKey" Type="Collection(Edm.PropertyPath)" Nullable="false">
           <Annotation Term="Core.Description" String="The set of primitive properties that identify a temporal object" />
-          <Annotation Term="Core.LongDescription" String="A temporal object is a set of facts whose change over application and/or system time is tracked by the service. The entities of the annotated entity set are time slices of temporal objects. A temporal object is uniquely identified within this set by the values of the specified object key properties. Object key properties follow the same rules as entity key properties. If no object key is specified, the object key is identical to the entity key." />
+          <Annotation Term="Core.LongDescription" String="A temporal object is a set of facts whose change over application and/or system time is tracked by the service. The entities of the annotated collection are time slices of potentially multiple temporal objects. A temporal object is uniquely identified within this set by the values of the specified object key properties. Object key properties follow the same rules as entity key properties. If no object key is specified, the annotated collection only contains time slices for a single one temporal object." />
         </Property>
         <Property Name="SupportedActions" Type="Collection(Temporal.QualifiedActionName)" Nullable="false">
           <Annotation Term="Core.Description" String="List of supported temporal actions" />
         </Property>
       </ComplexType>
 
-      <ComplexType Name="TimeDimensionType" Abstract="true">
-        <Property Name="SupportedQueries" Type="Temporal.QueryType" Nullable="false" />
+      <ComplexType Name="CollectionKindType" Abstract="true">
+        <Annotation Term="Core.Description" String="Kind of temporal collection" />
       </ComplexType>
 
-      <TypeDefinition Name="QualifiedActionName" UnderlyingType="Edm.String">
-        <Annotation Term="Core.Description" String="The qualified name of an action in scope." />
-        <Annotation Term="Core.LongDescription" String="*TODO: move this to Core, where we already have QualifiedTypeName and QualifiedTermName?*" />
-      </TypeDefinition>
+      <ComplexType Name="CollectionKindSnapshot" BaseType="Temporal.CollectionKindType">
+        <Annotation Term="Core.Description" String="Each OData entity corresponds to a temporal object and by default only shows data of the current time slice" />

```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="Each OData entity corresponds to a temporal object and by default only shows data of the current time slice" />
        <Annotation Term="Core.LongDescription" String="Each OData entity is uniquely identified by its [`ApplicationTimeSupport/ObjectKey`](#ApplicationTimeSupportType)" />
```
I want to make the relation between object key and (logical) entity key more explicit. A similar statement for time-slice collections is introduced with #128.

Reply:
Moved property `ObjectKey` to type `CollectionKindTimeSlice` because it is only relevant there.